Packs/Core/Playbooks/playbook-External_Login_Password_Spray.yml

```diff
@@ -0,0 +1,1474 @@
+    view: |-
+      {
+        "position": {
+          "x": 470,
+          "y": 3240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 19528476-25f9-4e0a-8827-107e4b0e45e0
+    type: title
+    task:
+      id: 19528476-25f9-4e0a-8827-107e4b0e45e0
+      version: -1
+      name: Soft Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "63"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 2140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: f1633e74-6dbb-453d-8490-b6679b6227b5
+    type: title
+    task:
+      id: f1633e74-6dbb-453d-8490-b6679b6227b5
+      version: -1
+      name: Hard Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "81"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 40,
+          "y": 2130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "45":
+    id: "45"
+    taskid: 119f0216-cd78-4fbc-81d8-3fc9e075d9fa
+    type: regular
+    task:
+      id: 119f0216-cd78-4fbc-81d8-3fc9e075d9fa
+      version: -1
+      name: Check reputation of Source IP
+      description: Enriches the external source IP of the attack to check if it's known as malicious. Skips on errors for cases where the IP address or the !ip command is empty.
+      script: '|||ip'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "46"
+    scriptarguments:
+      ip:
+        simple: ${alert.localip}
+    reputationcalc: 2
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -155
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "46":
+    id: "46"
+    taskid: a1924082-41b3-47f8-8766-d268b29a9658
+    type: condition
+    task:
+      id: a1924082-41b3-47f8-8766-d268b29a9658
+      version: -1
+      name: Is the IP malicious?
+      description: Checks if the external Source IP is malicious (DBotScore above 2).
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "15"
```

Comment:
change  the flow so the playbook does not pause on the manual task, let the playbook continue